tests/dummy/app/templates/docs/atoms/au-badge.md

```diff
@@ -0,0 +1,23 @@
+# Badge
+
+---
+
+{{#docs-demo as |demo|}}
+  {{#demo.example name='au-Badge.hbs'}}
+    <AuBadge @icon="info-circle"></AuBadge>
+    <hr>
+    <AuBadge @icon="info-circle" @skin="info"></AuBadge>
```

Comment:
Is it possible to self-close?

`<AuBadge @icon="info-circle" @skin="info" />`

Reply:
Yes